concepts/05 UI Components/TreeView/00 Getting Started with TreeView/70 Enhance Performance on Large Datasets.md

```diff
@@ -0,0 +1,7 @@
+If you use a large remote dataset, you can make the TreeView load data as a user expands nodes and thus enhance performance. To do this, enable the [Virtual Mode](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeView/VirtualMode/) or specify the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function as shown in the [Load Data on Demand](/Demos/WidgetsGallery/Demo/TreeView/LoadDataOnDemand) demo.
```

Comment:
Rename to Load Data on Demand

```suggestion
If the TreeView is bound to a large dataset, the component can load data and render nodes on demand as a user expands them. Because this tutorial uses only a small dataset, this functionality is not demonstrated here. You can refer to the following demos for information on how to configure it: [Virtual Mode](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeView/VirtualMode/) and [Load Data on Demand](/Demos/WidgetsGallery/Demo/TreeView/LoadDataOnDemand).
```